config/initializers/rubyzip.rb

```diff
@@ -0,0 +1,7 @@
+# https://github.com/rubyzip/rubyzip#configuration
+# Zip.setup do |c|
+#   c.on_exists_proc = true
+#   c.continue_on_exists_proc = true
+#   c.unicode_names = true
+#   c.default_compression = Zlib::BEST_COMPRESSION
+# end
```

Comment:
I don't think we need this, we can always add it if we do want to configure it ;)